docs/modules/user-guide/attachments/api-reference/next/index.html

```diff
@@ -900,7 +900,7 @@ <h3 class="panel-title">Devfile schema - Version 2.2.0-alpha</h3>
                                                                                                                                                                                                         <span class="json-property-required"></span>
                                                                                                                                                                                                 </dt>
                                                                                                                                                                                                 <dd>
-                                                                                                                                                                                                    <p>The port number should be unique.</p>
+                                                                                                                                                                                                    <p>Port number to be used within the container component. The same port cannot be used by two different container components.</p>
```

Comment:
Does this HTML file belong in the PR?

Reply:
The HTML is auto-generated by the build process and used in the docs.

I am unsure why it is causing whitespace differences, but I am assuming that any future PRs I make will not have this whitespace issue, so there won't be any diffs and then no corresponing commit.

I will check with Jake if I can just ignore these changes.

Reply:
If this is generated output, it might make sense to add an entry to the `.gitignore` file so it doesn't show up in your PRs.

Reply:
It is generated, but it is used by the docs build (like autogenerated API docs), so I don't think it can go in the .gitignore file as that would mean excluding it permanently.